### ISSUE TITLE
UnsatisfiedLinkError exception in deallocator for org.bytedeco.javacpp.Pointer.trimMemory()

### ISSUE THREAD

**Issue by rezangit** [bytedeco/javacv#874]
Hi

I got this exception during loading images, I have a big list of images to load and resize so this is happening after a heavy load. 

```
Mat buf = new Mat(imageData);
orgImg = imdecode(buf, IMREAD_COLOR);
resize(orgImg,destImg,size);
...
```

java.lang.UnsatisfiedLinkError: org.bytedeco.javacpp.Pointer.trimMemory()Z
	at org.bytedeco.javacpp.Pointer.trimMemory(Native Method)
	at org.bytedeco.javacpp.Pointer.deallocator(Pointer.java:558)
	at org.bytedeco.javacpp.Pointer.init(Pointer.java:121)
	at org.bytedeco.javacpp.BytePointer.allocateArray(Native Method)
	at org.bytedeco.javacpp.BytePointer.<init>(BytePointer.java:98)
	at org.bytedeco.javacpp.BytePointer.<init>(BytePointer.java:71)
	at org.bytedeco.javacpp.opencv_core$Mat.<init>(opencv_core.java:15518)
	at org.bytedeco.javacpp.opencv_core$Mat.<init>(opencv_core.java:15517)
	at com.bbyc.daps.integration.media.imageutil.ImageModifierOpenCV.<init>(ImageModifierOpenCV.java:24)

so based on my understanding, this happens in Pinter.deallocator(Deallocator deallocator) function, line 558

```
try {
while (count++ < maxRetries && ((maxBytes > 0 && DeallocatorReference.totalBytes + r.bytes > 
        maxBytes) || (maxPhysicalBytes > 0 && (lastPhysicalBytes = physicalBytes()) > maxPhysicalBytes))) {
        if (logger.isDebugEnabled()) {
               logger.debug("Calling System.gc() and Pointer.trimMemory() in " + this);
         }
         // try to get some more memory back
         System.gc();
         Thread.sleep(100);
          trimMemory();
     }
} catch (InterruptedException ex) {
        // reset interrupt to be nice
        Thread.currentThread().interrupt();
        } catch (UnsatisfiedLinkError e) {
        // old binaries with physicalBytes() or trimMemory() missing -> ignore for backward compatibility
        if (logger.isDebugEnabled()) {
        logger.debug(e.getMessage());
       }
}
```

that trimMemory() function is new and is not exist in binary files, so do I have a wrong version of binary files?
I only have this in my pom.xml file, is this enough to load the correct and update binary files?
I get a same error in MAC and Linux envirements.

```
<dependency>
	<groupId>org.bytedeco</groupId>
	<artifactId>javacv-platform</artifactId>
	<version>1.3.3</version>
</dependency>
```

**Comment by saudet** [comment]
It's a new function not present in old binaries, yes, but it's optional, the error gets caught:
https://github.com/bytedeco/javacpp/blob/master/src/main/java/org/bytedeco/javacpp/Pointer.java#L563

**Comment by rezangit** [comment]
so you mean javacv-platform 1.3.3 is coming with old binary?

**Comment by saudet** [comment]
No, but like I said, it doesn't matter. Please just ignore the error.

**Comment by saudet** [comment]
JavaCV 1.3.3 is the one that pulls old binaries, JavaCPP itself doesn't have binaries.

**Comment by saudet** [comment]
In any case, please use the just released version 1.4. The binaries now implement all the functions.
